doc/release/technotes/README.chpldoc

```diff
@@ -83,6 +87,10 @@ By default, documentation elements will be emitted in code order.  If
 the --docs-alphabetical flag is used, they will be sorted into
 alphabetical order within each lexical scope.
 
+If you would like a particular symbol to not be output as part of your
+documentation, you can specify this by prefacing the symbol with the
+pragma "no doc".
```

Comment:
Since we don't really define pragmas for the end-user anywhere (and they aren't intended for the end-user, really), you might want to put an example of what this looks like.  I was going to suggest pulling the sentence, but I think until we have private, it's probably a necessary evil to advertise to users.